apps/native-component-list/screens/GoogleScreen.js

```diff
@@ -19,13 +20,7 @@ export default class GoogleLoginScreen extends React.Component {
   _testGoogleLogin = async () => {
     try {
       const result = await Google.logInAsync({
-        androidStandaloneAppClientId:
-          '603386649315-87mbvgc739sec2gjtptl701ha62pi98p.apps.googleusercontent.com',
-        androidClientId: '603386649315-9rbv8vmv2vvftetfbvlrbufcps1fajqf.apps.googleusercontent.com',
-        iosStandaloneAppClientId:
-          '603386649315-1b2o2gole94qc6h4prj6lvoiueq83se4.apps.googleusercontent.com',
-        iosClientId: '603386649315-vp4revvrcgrcjme51ebuhbkbspl048l9.apps.googleusercontent.com',
-        scopes: ['profile', 'email'],
+        clientId: '603386649315-vp4revvrcgrcjme51ebuhbkbspl048l9.apps.googleusercontent.com',
```

Comment:
what happened to the androidClientId? does this one now work on both platforms? just want to make sure we can continue to test this everywhere.

Reply:
We only use the web login now.

Reply:
*facepalm*